localtypings/pxtarget.d.ts

```diff
@@ -146,6 +146,7 @@ declare namespace pxt {
         simAnimationEnter?: string;
         simAnimationExit?: string;
         projectGallery?: string;
+        gettingStarted?: string;
```

Comment:
There is a "sideDocs" property already. Don't we want to reuse or remove that one?